en/5x/api.md

```diff
@@ -1,6 +1,7 @@
 ---
 layout: 5x-api
 title: Express 5.x - API Reference
+description: Access the comprehensive API reference for Express.js 5.x, detailing all modules, methods, and properties for building web applications with this latest version.
```

Comment:
Access the API reference for Express.js 5.x, detailing all modules, methods, and properties for building web applications with this latest version.